runtime/dancebox/tests/common/xcm/constants.rs

```diff
@@ -218,11 +218,12 @@ pub mod frontier_template {
 
     pub fn genesis() -> sp_core::storage::Storage {
         let genesis_config = container_chain_template_frontier_runtime::RuntimeGenesisConfig {
+            system: Default::default(),
             balances: container_chain_template_frontier_runtime::BalancesConfig {
                 balances: pre_funded_accounts()
                     .iter()
                     .cloned()
-                    .map(|k| (k, 1 << 80))
```

Comment:
Why this?

Reply:
Seems to be a side effect of the new chain spec builder working with serde_json, as with anything bigger than a u64 it errors with with "number out of range"

Reply:
That has an easy fix, I saw it in the chain spec PR in polkadot-sdk, basically you need to add a feature to `serde_json` dependency.